docs/api/permissions.rst

```diff
@@ -0,0 +1,521 @@
+    HTTP/1.1 200 OK
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length, Last-Modified, ETag
+    Content-Length: 187
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 09:59:30 GMT
+    Etag: "1437040770742"
+    Last-Modified: Thu, 16 Jul 2015 09:59:30 GMT
+    Server: waitress
+
+    {
+        "data": {
+            "id": "46524be8-0ad7-3ac6-e260-71f8993feffa", 
+            "last_modified": 1437040770742
+        }, 
+        "permissions": {
+            "write": [
+                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
+            ]
+        }
+    }
+
+
+We can look at our own tasks collections permissions.
+
+.. code-block::
+
+    $ http GET http://localhost:8888/v1/buckets/default/collections/tasks --auth user:pass
+    HTTP/1.1 200 OK
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length, Last-Modified, ETag
+    Content-Length: 156
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 10:00:30 GMT
+    Etag: "1437040830468"
+    Last-Modified: Thu, 16 Jul 2015 10:00:30 GMT
+    Server: waitress
+
+    {
+        "data": {
+            "id": "tasks", 
+            "last_modified": 1437040830468
+        }, 
+        "permissions": {
+            "write": [
+                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
+            ]
+        }
+    }
+
+
+Managing object permissions
+===========================
+
+You can use PUT or PATCH to update or replace :term:`ACLs <ACL>` for an object.
+
+.. note::
+
+    The user that run the permission update is always added to the
+    write ACL in order not to loose control of the object.
+
+A blog bucket could be created with the following to give read access to everyone.
+
+.. code-block::
+
+    $ echo '{"data":{}, "permissions": {"read": ["system.Authenticated"]}}' | \
+        http PUT https://kinto.dev.mozaws.net/v1/buckets/servicedenuages-blog \
+        --auth user:pass
+
+    HTTP/1.1 201 Created
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length
+    Connection: keep-alive
+    Content-Length: 203
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 14:20:37 GMT
+    ETag: "1437056437581"
+    Last-Modified: Thu, 16 Jul 2015 14:20:37 GMT
+    Server: nginx/1.4.6 (Ubuntu)
+
+    {
+        "data": {
+            "id": "servicedenuages-blog", 
+            "last_modified": 1437056437581
+        }, 
+        "permissions": {
+            "read": [
+                "system.Authenticated"
+            ], 
+            "write": [
+                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
+            ]
+        }
+    }
+
+Then it is possible to create ``articles`` and ``comments`` collection
+with everyone having read access to their respective records.
```

Comment:
Ah ok.

Now, it will be possible to create two collections (``articles`` and ``comments``) in this bucket. We will give users the ``read`` permission on their own records.

See - link to use case doc -